client/components/upgrades/google-apps/google-apps-dialog/index.jsx

```diff
@@ -176,15 +133,21 @@ class GoogleAppsDialog extends React.Component {
 
 		return (
 			<footer className="google-apps-dialog__footer">
-				{ this.maybeShowKeepSearching() }
-				{ this.checkoutButtonOrLink() }
-				<button
-					className="google-apps-dialog__continue-button button is-primary"
-					type="submit"
+				{ ! this.state.isAddingEmail && (
+					<Button
+						className="google-apps-dialog__checkout-button"
+						onClick={ this.handleFormCheckout }
+					>
+						{ translate( 'Checkout' ) }
```

Comment:
Would `Proceed to Checkout` or `Skip this step` be more appropriate here? :thinking:

Reply:
I'd reuse the copy from before. Something along the:
```
No thanks, I don't need email`
```

/cc @ranh if you want to review/update something.

Reply:
I don't mind that one either, it might make the button too large though.